script.js

```diff
@@ -0,0 +1,156 @@
+    item.className = 'item ';
+    item.className += isItToDo;
+
+    const itemContent = document.createElement('div');
+    itemContent.className = 'item-content';
+    itemContent.innerHTML = content;
+
+    const radioButton = document.createElement('div');
+    radioButton.className = 'radio-button';
+
+    const itemRemove = document.createElement('button');
+    itemRemove.className = 'item-remove';
+
+    const itemContentBox = document.createElement('div');
+    itemContentBox.className = 'item-content-box';
+
+    itemContentBox.appendChild(radioButton);
+    itemContentBox.appendChild(itemContent);
+
+    item.appendChild(itemContentBox);
+    item.appendChild(itemRemove);
+
+    if (isItToDo === 'todo') {
+        todoList.appendChild(item);
+    } else {
+        doneList.appendChild(item);
+    }
+    countTodo();
+
+    //삭제 버튼 함수
+    const removeTodo = () => {
+        if (isItToDo === 'todo') {
+            todoArr = todoArr.filter((v) => v !== content);
+        }
+        if (isItToDo === 'done') {
+            doneArr = doneArr.filter((v) => v !== content);
+        }
+        item.remove();
+        syncLocalStorage();
+        countTodo();
+        toast('삭제되었어요!');
+    };
+    itemRemove.onclick = removeTodo;
+
+    //할일 토글 함수
+    const toggleTodo = () => {
+        if (isItToDo === 'todo') {
+            todoArr = todoArr.filter((v) => v !== content);
+            doneArr.push(content);
+        }
+        if (isItToDo === 'done') {
+            doneArr = doneArr.filter((v) => v !== content);
+            todoArr.push(content);
+        }
+
+        //현재 배열을 로컬스토리지에 최신화합니다.
+        syncLocalStorage();
+
+        // 화면에 보이는 요소들을 모두 지우고, 다시 로컬스토리지에서 불러옵니다.
+        while (todoList.firstChild) {
+            todoList.removeChild(todoList.lastChild);
+        }
+        while (doneList.firstChild) {
+            doneList.removeChild(doneList.lastChild);
+        }
+        getLocalStorage();
+    };
+    itemContent.onclick = toggleTodo;
+    radioButton.onclick = toggleTodo;
+};
+
+//엔터키로 추가 가능
+const enterPush = () => {
+    if (window.event.keyCode == 13) {
+        pushTodo();
+    }
+};
+
+//새 할일 추가하기 함수
+const pushTodo = () => {
+    const input = document.getElementById('input-form');
+    if (todoArr.indexOf(input.value) == -1 && doneArr.indexOf(input.value) == -1) {
+        //해당하는 값이 없을때만 추가!! : 이름이 같으면 토글할때 문제가 생기던데
+        if (input.value.length > 0) {
+            todoArr.push(input.value);
+            window.localStorage.setItem('todo', JSON.stringify(todoArr));
+            createTodoElement(input.value, 'todo');
+        }
+    } else {
+        toast('동일한 내용으론 작성할 수 없어요!');
+    }
+    input.value = null;
+};
+
+//투두 개수 카운팅
+const countTodo = () => {
+    const todo = document.getElementById('todo-count');
+    const done = document.getElementById('done-count');
+    todo.innerHTML = '(' + todoArr.length + ')';
+    done.innerHTML = '(' + doneArr.length + ')';
```

Comment:
이 부분에도 템플릿 리터럴을 사용해 볼 수 있을 것 같아요
```suggestion
    todo.innerHTML = `(${todoArr.length})`;
    done.innerHTML = `(${doneArr.length})`;
```